backend/battling/tests/test_view.py

```diff
@@ -0,0 +1,85 @@
+from django.test import Client, TestCase
+
+from model_bakery import baker
+
+from battling.models import PokemonTeam
+from pokemon.models import Pokemon
+from users.models import User
+
+
+class ListBattlesTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create(email="eduardavercosa@vinta.com.br", password="admin")
+        self.user.set_password("admin")
+        self.user.save()
+
+    def test_create_battle(self):
+        self.client.login(username=self.user.email, password="admin")
+        battles = baker.make("battling.Battle", creator=self.user, _quantity=2)
+        response = self.client.get("/battle/list/")
+        response_qs = response.context_data.get("battle_list")
+        self.assertCountEqual(battles, response_qs)
+
+
+class CreatePokemonTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create(email="eduardavercosa@vinta.com.br", password="admin")
+        self.user.set_password("admin")
+        self.user.save()
+
+        self.battle = baker.make("battling.Battle", creator=self.user, _quantity=1)
+        self.team = baker.make(
+            "battling.Team", battle=self.battle[0], trainer=self.user, _quantity=1
+        )
+
+        self.pokemon_1 = Pokemon.objects.create(
+            poke_id=18,
+            name="pidgeot",
+            img_url="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/18.png",  # noqa
+            attack=70,
+            defense=70,
+            hp=101,
+        )
+        self.pokemon_2 = Pokemon.objects.create(
+            poke_id=16,
+            name="pidgey",
+            img_url="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/16.png",  # noqa
+            attack=35,
+            defense=35,
+            hp=56,
+        )
+        self.pokemon_3 = Pokemon.objects.create(
+            poke_id=17,
+            name="pidgeotto",
+            img_url="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/17.png",  # noqa
+            attack=50,
+            defense=50,
+            hp=71,
+        )
+
+        self.pokemon_1.save()
+        self.pokemon_2.save()
+        self.pokemon_3.save()
```

Comment:
Note that when you use `Pokemon.objects.create` you don't need to call the save method after